src/auth/authData.ts

```diff
@@ -0,0 +1,11 @@
+import { z } from "zod";
+
+export const authJWTToken = z.object({
+  organizationId: z.string().uuid(),
+});
+export type AuthJWTToken = z.infer<typeof authJWTToken>;
+
+export const authData = z.object({
+  organizationId: z.string().uuid(),
+});
+export type AuthData = z.infer<typeof authData>;
```

Comment:
Why is this needed in addition to `authJWTToken` / `AuthJWTToken`?

(Also, not sure, but maybe it's more usual to capitalize codec names, so `AuthJWTToken` for both codec and type, so you can import both together?)

Reply:
Discussed offline: the point is that `AuthData` is expected to diverge from `AuthJWTToken` later to integrate more context data